ui/src/templates/constants/defaultTemplates.ts

```diff
@@ -397,28 +397,6 @@ export const localMetricsTemplate = () => ({
                   'from(bucket: v.bucket)\n  |> range(start: v.timeRangeStart, stop: v.timeRangeStop)\n  |> filter(fn: (r) => r._measurement == "boltdb_reads_total" or r._measurement == "boltdb_writes_total")\n  |> filter(fn: (r) => r._field == "counter")\n  |> derivative(unit: v.windowPeriod, nonNegative: true)\n  |> drop(columns: ["_field"])\n  |> yield(name: "derivative")',
                 editMode: 'advanced',
                 name: '',
-                builderConfig: {
```

Comment:
actually just removing them doesn't work they need to be replaced with a default builderConfig:                `builderConfig: {
                  buckets: [],
                  tags: [{key: '_measurement', values: []}],
                  functions: [],
                },`